clarifai/client/app.py

```diff
@@ -618,6 +618,116 @@ def patch_dataset(self, dataset_id: str, action: str = 'merge', **kwargs) -> Dat
 
     return Dataset.from_auth_helper(self.auth_helper, dataset_id=dataset_id, **kwargs)
 
+  def patch_model(self, model_id: str, action: str = 'merge', **kwargs) -> Model:
+    """Patches a model of the app.
+
+    Args:
+        model_id (str): The model ID of the model to patch.
+        action (str): The action to perform on the model (merge/overwrite/remove).
+        **kwargs: Additional keyword arguments to be passed to patch the Model.
+
+    Returns:
+        Model: A Model object of the specified model ID.
+    """
+    if "visibility" in kwargs:
+      kwargs["visibility"] = resources_pb2.Visibility(gettable=kwargs["visibility"])
+    if "image_url" in kwargs:
+      kwargs["image"] = resources_pb2.Image(url=kwargs.pop("image_url"))
+    request = service_pb2.PatchModelsRequest(
+        user_app_id=self.user_app_id,
+        models=[resources_pb2.Model(id=model_id, **kwargs)],
+        action=action)
+    response = self._grpc_request(self.STUB.PatchModels, request)
+    if response.status.code != status_code_pb2.SUCCESS:
+      raise Exception(response.status)
+    self.logger.info("\nModel patched\n%s", response.status)
+    kwargs.update({
+        'model_id': model_id,
+    })
+
+    return Model.from_auth_helper(self.auth_helper, **kwargs)
+
+  def patch_workflow(self, workflow_id: str, action: str = 'merge', **kwargs) -> Workflow:
```

Comment:
```suggestion
  def patch_workflow(self, workflow_id: str, action: str = 'merge', config_filepath: str = None,**kwargs) -> Workflow:
```

Can we add `config_filepath` param in the function param and add a description in Docstrings since this is kind of an main arg that'll be used and adding a description would help!

Reply:
Done